packages/certificate-summary/src/index.ts

```diff
@@ -65,6 +85,18 @@ function extractSummaryFromECoC(certificate: ECoCSchema): CertificateSummary | n
   const purchaseOrderPosition = StandardReferences?.find((ref) => ref?.name === 'OrderPos')?.Value;
   const purchaseDeliveryNumber = StandardReferences?.find((ref) => ref?.name === 'DeliveryNote')?.Value;
 
+  const ecocData = certificate.EcocData as EcocData;
+  const orderQuantityInKG = (
+    ecocData?.Data?.ObjectOfDeclaration?.reduce((acc, objOfDeclarartion) => {
+      return (
+        acc +
+        objOfDeclarartion?.Quantities?.reduce((acc, quantitiesObj) => {
+          return acc + (quantitiesObj?.Amount || 0);
+        }, 0)
+      );
+    }, 0) || ''
```

Comment:
this is a simplification of what could be in the E-CoC certificate as the unit is not always kilogram, right ?